src/04-modality-specific-files/01-magnetic-resonance-imaging-data.md

```diff
@@ -698,35 +714,60 @@ sub-<label>/[ses-<label>/]
-`k-`. For these differentially phase encoded sequences, one also needs to
-specify the Total Readout Time defined as the time (in seconds) from the center
-of the first echo to the center of the last echo (aka "FSL definition" - see
-[here](https://fsl.fmrib.ox.ac.uk/fsl/fslwiki/topup/Faq#How_do_I_know_what_phase-encode_vectors_to_put_into_my_--datain_text_file.3F) and
-[here](https://lcni.uoregon.edu/kb-articles/kb-0003) how to calculate it). For
-example
+The [`dir-<label>`](../99-appendices/09-entities.md#dir) entity is REQUIRED
+for these files.
+This key-value pair SHOULD NOT be used to replace the required `PhaseEncodingDirection`
+(nor any other acquisition parameter, as prescribed by the
+[common principles](../02-common-principles.md#file-name-structure)).
+
+Required fields:
+
+| **Key name**           | **Requirement level** | **Data type** | **Description**                                                                    |
+| ---------------------- | --------------------- | ------------- | ---------------------------------------------------------------------------------- |
+| PhaseEncodingDirection | REQUIRED              | [string][]    | See [in-plane spatial encoding](#in-plane-spatial-encoding) table of fields.       |
+| TotalReadoutTime       | REQUIRED              | [number][]    | See [in-plane spatial encoding](#in-plane-spatial-encoding) table of fields.       |
+
+For example:
 
 ```JSON
 {
    "PhaseEncodingDirection": "j-",
-   "TotalReadoutTime": 0.095,
-   "IntendedFor": "func/sub-01_task-motor_bold.nii.gz"
+   "TotalReadoutTime": 0.095
 }
 ```
 
-The `label` value of the [`dir-<label>`](../99-appendices/09-entities.md#dir) key/value pair
-can be set to arbitrary alphanumeric label (`[a-zA-Z0-9]+` for
-example `LR` or `AP`) that can help users to distinguish between different
-files, but should not be used to infer any scanning parameters (such as phase
-encoding directions) of the corresponding sequence. Please rely only on the JSON
-file to obtain scanning parameters. \_epi files can be a 3D or 4D - in the
-latter case all timepoints share the same scanning parameters. To indicate which
-run is intended to be used with which functional or diffusion scan the
-IntendedFor field in the JSON file should be used.
+As for other EPI sequences, these field mapping sequences may have any of the
+[in-plane spatial encoding](#in-plane-spatial-encoding) metadata keys.
+However, please note that `PhaseEncodingDirection` and `TotalReadoutTime` keys
+are REQUIRED for these field mapping sequences.
+
+### Expressing the MR protocol intent for fieldmaps
+
+Fieldmaps are typically acquired with the purpose of correcting one or more EPI
+scans under `func/` or `dwi/` for distortions derived from *B<sub>0</sub>*
+nonuniformity.
+This linking between fieldmaps and their targetted data MAY be encoded with the
+`IntendedFor` metadata.
+
+#### Using `IntendedFor` metadata
```

Comment:
I guess the reason for dropping `IntendedFor` from earlier examples is because it's now introduced afterwards? I think this is generally confusing, as it breaks up fields that are going to appear together.

I'm not sure whether we should push this back up to the introductory section or just add:

```Markdown
See [Using `IntendedFor` metadata](#using-intendedfor-metadata)
for details on the `IntendedFor` field.
```

After each example.

Reply:
> I guess the reason for dropping `IntendedFor` from earlier examples is because it's now introduced afterwards?

Correct. I've edited as suggested.